src/Orc.Automation/Extensions/AutomationElementExtensions.ElementTree.cs

```diff
@@ -2,10 +2,11 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Windows.Automation;
 using Controls;
 
-public static partial class AutomationElementExtensions
+public partial class AutomationElementExtensions
```

Comment:
I think extensions should be static?

Reply:
partial.. .static lost somewhere in other parts)... will add it here too